flipperdevices/flipperzero-firmware
# Picopass fix ice

# What's new

- Fixing issue from https://github.com/flipperdevices/flipperzero-firmware/issues/2834

# Verification 

- Read a card
- Choose "Change key"
- Write iCE 
- Check key is diversified using elite KDF or Use with iCopyX as iCE card

# Checklist (For Reviewer)
